.gitignore

```diff
@@ -1,3 +1,6 @@
 *~
 *.so
 /src/tools/simple-kernel-timer/kp_reader
+
+# Ignore the 'build' directory
+build/*
```

Comment:
Why would that be needed?
If it was up to me I would remove the file altogether.

Reply:
Should I remove it then ?